cmsdb/campaigns/run3_2022_preEE_nano_uhh_v12/hhh4b2tau.py

```diff
@@ -0,0 +1,180 @@
+
+import cmsdb.processes as procs
+from cmsdb.campaigns.run3_2022_preEE_nano_uhh_v12 import campaign_run3_2022_preEE_nano_uhh_v12 as cpn
+
+# SM sample
+cpn.add_dataset(
+    name="hhh_4b2tau_c3_0_d4_0_madgraph",
```

Comment:
regarding the process names, we decided to skip the underscore between the parameter and it's value (e.g. kl1 instead of kl_1). I see that this would become quite confusing with parameters that contain numbers in the parameter name, but this is already done in that way in other places, e.g. here:  https://github.com/uhh-cms/cmsdb/blob/master/cmsdb/processes/hh.py#L157


also, I'd abbreviate the `minus` with a `m`